modules/util/source-formatter/src/test/resources/com/liferay/source/formatter/dependencies/expected/upgrade/UpgradeCatchAllCheck.testjava

```diff
@@ -1307,6 +1309,9 @@ public class UpgradeCatchAllCheck implements ConfigurationBeanDeclaration {
 
 	@Reference
 	private CPInstanceService _cpInstanceService;
+	
+	@Reference
+	private DDMFormViewFormInstanceRecordsDisplayContext _ddmFormViewFormInstanceRecordsDisplayContext;
```

Comment:
@MicaelleSilvaa please create a method receiving this as a parameter. Being a parameter removes the "_" of the variable name.